lib/dao.js

```diff
@@ -1613,14 +1637,16 @@ DataAccessObject._coerce = function(where) {
           switch (operator) {
             case 'inq':
             case 'nin':
-              if (!Array.isArray(val)) {
+            case 'between':
+              try {
+                val = coerceArray(val);
+              } catch (e) {
```

Comment:
Ditto - please log `e`. I think it may be best to include it in the error message?

```js
debug('Cannot coerce array clause: ', e);
err = new Error(g.f('The %s property has invalid clause %j: %s', p, where[p], e.message));
```